docs/OMS-MySQL-Instructions.md

```diff
@@ -6,10 +6,10 @@
 
```

Comment:
Add a step here about acquiring the workspace ID for the workspace that they want the logs to be sent to.